solidity/contracts/bancorx/BancorX.sol

```diff
@@ -304,7 +310,33 @@ contract BancorX is Owned, TokenHolder, ContractIds {
         prevLockLimit = currentLockLimit.sub(_amount);
         prevLockBlockNumber = block.number;
 
-        emit XTransfer(msg.sender, _toBlockchain, _to, _amount);
+        // emit XTransfer event with id of 0
+        emit XTransfer(msg.sender, _toBlockchain, _to, _amount, 0);
+    }
+
+    /**
+        @dev claims BNT from msg.sender to be converted to BNT on another blockchain
+
+        @param _toBlockchain    blockchain BNT will be issued on
+        @param _to              address to send the BNT to
+        @param _amount          the amount to transfer
+        @param _id              id
```

Comment:
replace with a proper comment - get used to properly commenting arguments - if the description is identical to the argument name, you might as well not even put it there